build.gradle

```diff
@@ -34,11 +34,12 @@ description = 'New code for Egeria'
 
 // TODO: Specify Dependency versions as variables
 ext {
-    egeriaversion = '3.6'
+    egeriaversion = '3.10'
```

Comment:
egeria: note we're up to 3.14 currently
jackson: 2.14.2
jupiter 5.0.2
httpclient 4.5.14
mockito 4.11.0 (also to v5)

One option may be that most of these can be removed (except egeria) and then import the bom ie
```
         implementation platform("org.odpi.egeria:egeria:${egeriaversion}")
```

This way you should pick up the version constraints of egeria, which I would recommend as the starting point to avoid conflicts (since we don't use sharding or isolating class loaders)